apps/jackbnb-fe/src/components/SpaceType/index.module.css

```
@@ -0,0 +1,19 @@
+
+.myContainer{
+  overflow-x:hidden;
+}
+
+.myContainer_items {
+    display:grid;
+    grid-template-columns:repeat(5,121px);
+    width:98%;
+    grid-gap:10px;
```

Please try to remember respect the spaces in the CSS rules. Let your code breath
```suggestion
    display: grid;
    grid-template-columns: repeat(5, 121px);
    width: 98%;
    grid-gap: 10px;
```